### ISSUE TITLE
feature map normalization 시 0으로 나누어 NaN이 되는 오류 발생

### ISSUE THREAD

**Issue by jsleeg98** [jsleeg98/KD_PaT#8]
![image](https://user-images.githubusercontent.com/64693974/212851991-34e0918f-fb96-4877-bbe9-bedfecf29e68.png)

9b7e6328158c10ad920063e5470959a9e237885f

**Comment by jsleeg98** [comment]
torch.where 을 활용하여 0인 값을 1로 바꾸어 해결
